tech-matters-serverless-helpers-lib/tests/helpers.ts

```diff
@@ -0,0 +1,48 @@
+const Twilio = require('twilio');
```

Comment:
Can we name this file something a little more descriptive? Like `twilioGlobals.ts` or something? 

Sorry to nitpick - modules with generic names like `helper.ts` or `util.ts` are my pet hate :-P

Reply:
That makes sense, thanks for the feedback :P https://github.com/techmatters/serverless/pull/315/commits/0fb99be6cfed7e03b4fb1c6e47d12757ecab99f9